examples/deployment-with-init/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployment Example
+
+This example shows how to deploy an Elastic Cloud deployment using Terraform.
+It will also initialize the instance by using some of the outputs as string variables within a very simple bash script, to create a user and an index.
```

Comment:
```suggestion
This example shows how to create an Elastic Cloud deployment using Terraform.

Secondly, it will show you how to initialize the instance. This is done by using some of the outputs as string variables within a very simple bash script to create a user and an index.
```